pkgs/dart_mcp/lib/src/api/initialization.dart

```diff
@@ -194,6 +194,7 @@ extension type ServerCapabilities.fromMap(Map<String, Object?> _value) {
     Prompts? prompts,
     Resources? resources,
     Tools? tools,
+    @Deprecated('Do not use, only clients have this capability')
```

Comment:
Oops.  I thought the server had that in its schema to indicate that it might send elicitations.